src/services/pastPurchases.ts

```diff
@@ -0,0 +1,22 @@
+import { Events } from '@storefront/flux-capacitor';
+import { core, BaseService } from '../core/service';
+import StoreFront from '../storefront';
+
+@core
+class PastPurchaseService extends BaseService<PastPurchaseService.Options> {
+
+  constructor(app: StoreFront, opts: any) {
+    super(app, opts);
+    this.app.flux.on(Events.AUTOCOMPLETE_QUERY_UPDATED, () => this.app.flux.actions.receiveSaytPastPurchases([]));
+  }
+
+  init() {
+    // no-op
```

Comment:
Is this still needed

Reply:
You need to have an init method since it extends a class with an abstract init method, but we don't need a definite init here, so we just have a no-op. This is also done in the search service.